packages/valory/skills/decision_maker_abci/redeem_info.py

```diff
@@ -134,7 +134,7 @@ def is_winning(self) -> bool:
     @property
     def claimable_amount(self) -> int:
         """Get the claimable amount of the current market."""
-        amount = int(self.outcomeTokenMarginalPrice * self.outcomeTokensTraded)
+        amount = self.outcomeTokensTraded
```

Comment:
The previous calculation was resulting in the number of conditional tokens instead of the amount in WEI.